packages/ui/src/primitives/tailwind/Radio/index.stories.tsx

```diff
@@ -23,29 +23,92 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
 Infinite Reality Engine. All Rights Reserved.
 */
 
-import Component from './index'
+import React, { useEffect, useState } from 'react'
+import { ArgTypes } from 'storybook/internal/types'
+import RadioGroup, { RadioProps } from './index'
+
+const argTypes: ArgTypes = {
+  disabled: {
+    control: 'boolean'
+  },
+  selected: {
+    control: 'boolean',
+    name: 'Initially selected'
+  },
+  label: {
+    control: 'text'
+  },
+  description: {
+    control: 'text'
+  },
+  horizontal: {
+    control: 'boolean'
+  },
+  numberOfRadios: {
+    control: 'number',
+    name: 'Number of Radios',
+    description: 'The number of radios to generate'
+  }
+}
 
 export default {
   title: 'Primitives/Tailwind/Radio',
-  component: Component,
+  component: RadioGroup,
   parameters: {
     componentSubtitle: 'Radio',
-    jest: 'Radio.test.tsx',
     design: {
       type: 'figma',
-      url: ''
+      url: 'https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2786-21237&node-type=frame&t=n2wfUzH1bzwYsyob-0'
     }
+  },
+  argTypes,
+  args: {
+    numberOfRadios: 1,
+    label: 'Remember me',
+    description: 'Save my login details for next time'
   }
 }
 
+const RadioGroupRenderer = ({
+  description,
+  label,
+  disabled,
+  numberOfRadios,
+  horizontal,
+  selected
+}: Pick<RadioProps, 'description' | 'label' | 'disabled'> & {
+  numberOfRadios: number
+  horizontal: boolean
+  selected: boolean
+}) => {
+  const options =
+    numberOfRadios > 1
+      ? Array.from({ length: numberOfRadios }, (_, idx) => ({
+          label: `${label} ${idx + 1}`,
+          description: `${description} ${idx + 1}`,
+          value: `${idx + 1}`
+        }))
```

Comment:
Let's have a story which has description unset completely. Currently when there are multiple radios with description set to empty, we still see index as description text.
![image](https://github.com/user-attachments/assets/e9a28f63-4db4-4048-859c-101406d452b9)